asciidoctorj-core/src/main/java/org/asciidoctor/ast/AbstractBlockImpl.java

```diff
@@ -71,45 +61,25 @@ public Object content() {
 
     @Override
     public Object getContent() {
-        return delegate.content();
-    }
-
-    @Override
-    public String getNodeName() {
-        return delegate.getNodeName();
+        return JavaEmbedUtils.rubyToJava(getProperty("content"));
     }
 
     @Override
     public String convert() {
-        return delegate.convert();
+        return getString("convert");
     }
```

Comment:
Are you ok if we move this change? 
Because changing the type of this method has some other implications that lead away from what this PR wants to achieve.
See https://github.com/asciidoctor/asciidoctorj/blob/asciidoctorj-1.6.0/asciidoctorj-core/src/main/java/org/asciidoctor/internal/JRubyAsciidoctor.java#L213